plugins/filter_go/cgo.h

```diff
@@ -0,0 +1,51 @@
+/* Created by "go tool cgo" - DO NOT EDIT. */
```

Comment:
This header should be committed in repository?
It is unnatural that this header is committed this repo.
Because `Created by "go tool cgo" - DO NOT EDIT. ` says that this header will be automatically generated when building with cgo.
https://github.com/fluent/fluent-bit/pull/1277/files#diff-a620f3b51b8e26da1ae15c0b4335ca94ff816911ae0987b0297effa5f3db800aR45-R49 are needed in go filter?
WDYT?

Reply:
This file is build by cgo, But it is build by go project, not this project.